Project-Timer-frontend/src/app/dashboard/project-details/project-details.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input } from '@angular/core';
+import { Project } from '@app-interfaces/project.interface';
+import { FirebaseService } from '@app-services/firebase.service';
+
+@Component({
+  selector: 'app-project-details',
+  templateUrl: './project-details.component.html',
+  styleUrls: ['./project-details.component.scss'],
+})
+export class ProjectDetailsComponent {
+  @Input() project: Project;
+
+  constructor(private firebaseService: FirebaseService) {}
+
+  updateProjectName(projectName: string) {
+    this.firebaseService.updateProject(this.project.key, {
+      projectName: projectName,
+    });
+  }
+
+  deleteProject() {
+    this.firebaseService
+      .deleteProject(this.project.key)
+      .catch((err) => console.log(err));
```

Comment:
ten console.log niepotrzebny, jesli chcesz wyrzucić użytkownikowi zwrotkę z błędem to możesz wykorzystać np. 
https://material.angular.io/components/snack-bar/overview